planetary-social/nos
# #1600: Build Your Network!

## Issues covered
#1600 

## Description
Adds the Build Your Network screen to Onboarding after the Terms screen.

Note: adding an SVG failed (something was wrong with the file), so instead Sebastian gave me PNGs. He'll work on fixing the SVG and get that to me tomorrow.

## How to test
1. Log out
2. Navigate through onboarding
3. Observe Build your network!
4. Tap Find people button
5. Observe Discover tab

## Screenshots/Video

https://github.com/user-attachments/assets/d97c3d4a-9f41-46a8-8718-f70eb1ce0242
